pkg/rulerequest/handle.go

```diff
@@ -65,3 +79,47 @@ application. If you are unsure say no.`)
 	pterm.Println()
 	return prompt.Bool("Do you want to allow this app to have these (POTENTIALLY DANGEROUS) permissions?", false)
 }
+
+func handleNotAllowed(dangerous bool, image string) (string, error) {
+	if dangerous {
+		return "yes", nil
+	}
+
+	pterm.Warning.Printfln(
+		`This application would like to use the image '%s'.
+This could be VERY DANGEROUS to the cluster if you do not trust this
+application. If you are unsure say no.`, image)
```

Comment:
I think we should change this warning message. It would be rather alarming for a user who is unfamiliar with image allow rules and might make them think that there is something dangerous within the image itself (but it could be innocuous, like an image from our library repo).

This is what I would suggest:

```suggestion
		`This application would like to use the image '%s'.
This image is not trusted by any image allow rules in this project.
This could be VERY DANGEROUS to the cluster if you do not trust this
application. If you are unsure say no.`, image)
```